src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -667,7 +669,7 @@
   ;; Here, keywords represent a single element in the path. Anything between two '/' counts.
   ;; The second vector takes anything else that might be on the end of the path.
   ;; Below, this corresponds to '*/*/files/**' or '*/*/tasks/**' in a filesystem glob.
-  (bidi.bidi/match-route [[#"[^/]+/" :module-name #"/(files|tasks)/" [#".+" :rest]] :_]
+  (bidi.bidi/match-route [[#"[^/]+/" :module-name #"/(files|tasks|lib)/" [#".+" :rest]] :_]
```

Comment:
The error message on line 703 needs to be updated.
Can someone verify that this is ok from a security perspective since it's doesn't seem to align 100% with the plugins mount?